Add The First osu! Manga: osu! Combat Championship news post (#10142)
ppy/osu-wiki: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md

---
layout: post
title: "The First osu! Manga: osu! Combat Championship"
date: 2023-10-21 13:00:00 +0000
---

> WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted.

Reviewer: ```suggestion
WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted!
```

![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/OCCnewspostrecent.jpg)

**Calling all osu! enthusiasts!**

>  

Reviewer 2: ```suggestion

```
was a space in there

Prepare yourselves for a journey like none other, as we introduce the highly anticipated **osu! Combat Championship** (OCC) manga!

>  Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into our unique spin on the world of osu!.

Reviewer: ```suggestion
Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into our unique spin on the world of osu!.
```

For those eager to dive into the manga and skip the introductions, simply click the button below to head over to Mangadex and start reading.

<p align="center">
  <a href="https://mangadex.org/title/f1d50eba-6ace-4490-8439-07692fda3b9c/osu-combat-championship">
    <img src="/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/occbuttonnew.png" alt="Character Panels">
  </a>
</p>

Curious about which of your favorite players are in The osu! Combat Championship manga? Check out the character introductions below:

> [![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)

Reviewer: not sure what's going on with each banner linking to some kind of temporary photo. it'd be better to make them link to each player's osu! profile if you really want to make them clickable.

**same applies to other banners.**

### without link:
```suggestion
![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)
```
### with link:
```suggestion
[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)](https://osu.ppy.sh/users/4504101)
```

Reviewer 3: these are meant to link to full-size versions of the images

Author: Yeah lol

Author: > these are meant to link to full-size versions of the images

Yeah lol

Reviewer: will keep this unresolved until the temp images are replaced with open ones then

> WhiteCat: WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harbored blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.

Reviewer 2: I am not sure if the `player:` thing is necessary if you have the description under the banner and it mentions them by name
```suggestion
WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harboured blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.
```

[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/mrekkcard.png)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)

> mrekk: A true businessman, willing to go to any lengths to secure profits and success. He seldom finds time or inclination for frivolous pursuits, such as relationships or family, as his primary focus is on strengthening his business. His unwavering desire for profit is rooted in a profound fear of poverty, stemming from a childhood trauma that has left a lasting impact.

Reviewer 2: ```suggestion
A true businessman, mrekk is willing to go to any lengths to secure profits and success. He seldom finds time or inclination for frivolous pursuits, such as relationships or family, as his primary focus is on strengthening his business. His unwavering desire for profit is rooted in a profound fear of poverty, stemming from a childhood trauma that has left a lasting impact.
```

[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)

> Umbre: On the surface, he's a bit of a jokester, often cracking light-hearted quips at WhiteCat and creating a generally pleasant atmosphere. He serves as an exceptionally effective mentor, stepping into the role of WhiteCat's brother-figure after the loss of BlackDog and helping to nurture him into the state he's in today. Umbre suffers an internal struggle, torn between being Umbre the player or Umbre the brother. Nonetheless, his kind-hearted nature prevails, leading him to make the decision to guide WhiteCat to heights he could never have imagined.

Reviewer 2: ```suggestion
On the surface, Umbre is a bit of a jokester, often cracking light-hearted quips at WhiteCat and creating a generally pleasant atmosphere. He serves as an exceptionally effective mentor, stepping into the role of WhiteCat's brother-figure after the loss of BlackDog and helping to nurture him into the state he's in today. Umbre suffers an internal struggle, torn between being Umbre the player or Umbre the brother. Nonetheless, his kind-hearted nature prevails, leading him to make the decision to guide WhiteCat to heights he could never have imagined.
```

[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/kariyunewcard.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)

> Kariyu: An extroverted and playful individual. He revels in being a jokester and easily captivates a broad audience. As a child, he harbored a strong desire to emulate his idol, Vaxei. In pursuit of this dream, he embarked on a solo journey to the new continent and returned with a newfound companion, Cooper. Together, they have embarked on countless adventures and taken part in numerous tournaments.

Reviewer 2: ```suggestion
Kariyu is an extroverted and playful individual. He revels in being a jokester and easily captivates a broad audience. As a child, he harboured a strong desire to emulate his idol, Vaxei. In pursuit of this dream, he embarked on a solo journey to the new continent and returned with a newfound companion, Cooper. Together, they have embarked on countless adventures and taken part in numerous tournaments.
```

[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/xootycard.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)

> xootynator: An extroverted and composed individual, xootynator exudes boundless optimism. Her quest for enjoyment is inseparable from her determination to make her mark in the predominantly male-dominated arena of combat. Fueled by the excitement of battle and a commitment to shatter gender barriers, xootynator also finds immense joy in the world of art, with a particular fondness for birds.

Reviewer 2: ```suggestion
An extroverted and composed individual, xootynator exudes boundless optimism. Her quest for enjoyment is inseparable from her determination to make her mark in the predominantly male-dominated arena of combat. Fuelled by the excitement of battle and a commitment to shatter gender barriers, xootynator also finds immense joy in the world of art, with a particular fondness for birds.
```

> Meet WhiteCat, mrekk, and a bunch of other interesting characters in the OCC manga! Each one's got their own unique style and story. As you flip through those character panels, you're in for a diverse crew — some cool, some quirky, but all ready to bring their own flavor to the OCC manga!. It's a character buffet of people you already know, so get ready to dig in and enjoy the ride!

Reviewer 2: ```suggestion
Meet WhiteCat, mrekk, and a bunch of other interesting characters in the OCC manga! Each one's got their own unique style and story. As you flip through those character panels, you're in for a diverse crew — some cool, some quirky, but all ready to bring their own flavour to the OCC manga! It's a character buffet of people you already know, so get ready to dig in and enjoy the ride!
```

But that's not all! We have an exciting announcement for all anime enthusiasts out there. The OCC manga will be transformed into an **animated series**, and we have started on the first episode's production!

## The current team working on this project behind the scenes!

<p align="center">
  <img width="98%" height="98%" src="/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/occteamnew.png">
</p>

---

If you are a top 50 player and want to be a part of the series, or someone who wants to participate in the production of OCC, feel free to reach out to [VTX](https://osu.ppy.sh/users/17539772) on any platform. The team is open to collaborations and welcomes anyone with experience.

> Prepare yourselves for an unforgettable adventure as osu! Combat Championship takes center stage. Stay tuned for further updates on the manga!

Reviewer 2: ```suggestion
Prepare yourselves for an unforgettable adventure as osu! Combat Championship takes centre stage. Stay tuned for further updates on the manga!
```
